packages/two_percent/lib/two_percent/configuration.rb

```diff
@@ -30,6 +30,8 @@ module TwoPercent
   #   end
   #
   config_accessor :authenticate do
-    ->(*) { true }
+    ->(*) do
+      false
+    end
```

Comment:
If we configured a logger we could also add a message to help guide the user of the gem on how to configure this.

```suggestion
    ->(*) do
          TwoPercent.logger.warn(<<~MESSAGE)
          TwoPercent authenticate is currently configured using a default and is blocking authenticaiton.

          To make this wraning go away provide a configuration for `TwoPercent.config.authenticate`.
          The value should:
            1. Be callable like a Proc.
            2. Return true when the request is permitted.
            3. Return false when the request is not permitted.
        MESSAGE
        
      false
    end
```